std-bits/table/src/main/java/org/enso/table/data/column/builder/object/Builder.java

```diff
@@ -10,26 +10,26 @@
 public abstract class Builder {
   public static Builder getForType(StorageType type, int size) {
     Builder builder = switch (type) {
-      case AnyObjectType() -> new ObjectBuilder(size);
-      case BooleanType() -> new BoolBuilder(size);
-      case DateType() -> new DateBuilder(size);
-      case DateTimeType() -> new DateTimeBuilder(size);
-      case TimeOfDayType() -> new TimeOfDayBuilder(size);
-      case FloatType(Bits bits) ->
-        switch (bits) {
+      case AnyObjectType x -> new ObjectBuilder(size);
```

Comment:
Yup, this will prevent the [incremental compilation problems](https://github.com/eppleton/frgaal/issues/10) as matching on type doesn't need to recognize `record`.

Reply:
Yeah, I was sad to see the neat matching go, but better build stability is more important.